pulsar-discovery-service/src/test/java/org/apache/pulsar/discovery/service/DiscoveryServiceTest.java

```diff
@@ -90,9 +91,11 @@ private void clean() throws Exception {
     public void testBrokerDiscoveryRoundRobin() throws Exception {
         addBrokerToZk(5);
         String prevUrl = null;
+        BrokerDiscoveryProvider discoveryProvider = service.getDiscoveryProvider();
+        assertEquals(discoveryProvider.getAvailableBrokers().size(), 5);
```

Comment:
would there be a need to use Awaitility to make sure that there are 5 available brokers?

Reply:
the wait is done inside addBrokerToZk (that is the actual fix)
I added this line in order to double check in case of a new failure, 
I can remove it, is is basically redundant